Sources/Subs-Post.php

```diff
@@ -2972,7 +2974,8 @@ function spell_check($dict, $word)
  *
  * Returns an array of suggested replacements for the specified word
  *
- * @param resource $dict An enchant or pspell dictioary resource
+ * @param resource $dict An enchant or pspell dictionary
+ * resource
```

Comment:
Er, why is this on another line?

Reply:
It needed some space...

Reply:
Meh. I'll just merge and then adjust it myself. 🙂

Reply:
It wasn't moved, there was a typo in dictionary.